Governance/create/create-publishing.md

```diff
@@ -0,0 +1,13 @@
+# Requirements for publication
+
+## Publishing in a journal
+
+Move journal requirements here? https://acdguide.github.io/Governance/concepts/journal.html

```

Comment:
I would simply say something like "Many journals require data (and sometimes code) be published accompanying a manuscript. See <link to concepts/journal> for examples."